unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/Catalog/BIWCatalogManager.cs

```diff
@@ -9,17 +9,16 @@
 public static class BIWCatalogManager
 {
     public static bool VERBOSE = false;
-    private static bool INIT = false;
+    private static bool IS_INIT = false;
 
     public static void Init()
```

Comment:
Two concerns:

1. We have an `Init()` without a `Dispose()` here. 
2. If this class needs to track state (as intended with the use of `IS_INIT`) we should make it an instance class. Otherwise we risk having a bad state on test execution time.

Reply:
This class was originally a singleton , I moved to a static class because of our conventions to get rid of singletons. 

However, this class should live as long as the app is running since it will handle all the catalog/collectibles changes.

If we want to have a `Dispose()` method and make this class an instance class, we should refactor it to handle all the content from catalog/collectibles each time that initialize and init it each time with the builder in world which can have some performance trade-off. 

We can add a technical debt issue if you want